src/Model/Table/SentencesTable.php

```diff
@@ -191,7 +191,9 @@ public function beforeSave($event, $entity, $options)
                 $user = $this->Users->get($entity->user_id);
                 if ($user) {
                     $userDefaultLicense = $user->settings['default_license'];
-                    $entity->license = $userDefaultLicense;
+                    $entity->license = $entity->based_on_id == 0 ?
+                                       $userDefaultLicense :
+                                       'CC BY 2.0 FR';
```

Comment:
If `$entity->based_on_id` is null, this will set the license to `$userDefaultLicense`. It’s not supposed to happen on sentence creation, but who knows. Could you change this into triple equal just in case?

Reply:
Sure.